src/grt/src/GlobalRouter.cpp

```diff
@@ -33,6 +33,8 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
+#include "grt/GlobalRouter.h"
+
```

Comment:
This issue with moving header lines keeps popping up.  Is this due to a bad version of clang-format? This passes the format check.

Reply:
Not sure about that. In this PR, the format check complained about the header being down in the list, so I've run clang-format for this file. I', using `clang-format version 14.0.6` on a macos

Reply:
@vvbandeira I don't think this is desired behavior

Reply:
What is the concern?

Reply:
Different versions of clang-format keep changing the position of the header files.

* Here the header is moved to the top for some reason.
* 72b7afd moves `#include "FastRoute.h"` to alphabetical order.
* dc8d7ec moves `#include "sta/Parasitics.hh"` to alphabetical order.
* c78920b moves `#include "ppl/IOPlacer.h"` to the top

There's been more instances but those are the ones I found in the last 50 PRs or so.

I would think keeping the headers in alphabetical order makes more sense, no?

Reply:
clang-format splits includes into groups and sorts within groups.  In google style, which we use, <> and "" are separate groups.   The first two are as expected but the third seems odd.   @eder-matheus what version of clang-format are you using?

Once this check becomes mandatory it will be an error to use a non-conforming version.  The action uses 14.0.6

Reply:
The weirdness you see in c78920b is the same weirdness in this commit, and it's what the format CI wants (see [the CI failure](https://github.com/The-OpenROAD-Project/OpenROAD/actions/runs/3298813598/jobs/5441411670)). So this problem exists in the current format CI.

Reply:
Its a good practice that foo.cc include foo.h first to validate that the header is correct without relying on any further headers to be included first.  I didn't realize clang-format does that but I am in favor of it.  Other headers are grouped separately and sorted.

Reply:
gotcha